lib/operation.js

```diff
@@ -1,14 +1,18 @@
 'use strict';
 
+const color = require('color');
 const is = require('./is');
 
 /**
  * Rotate the output image by either an explicit angle
  * or auto-orient based on the EXIF `Orientation` tag.
  *
- * If an angle is provided, it is converted to a valid 90/180/270deg rotation.
+ * If an angle is provided, it is converted to a valid positive degree rotation.
  * For example, `-450` will produce a 270deg rotation.
  *
+ * If a non-rectangular angle is provided, the color of the background color
```

Comment:
I think the mathematical term would be "non-quadrantal angle" but maybe "angles that are not multiple of 90" would be better understood.